data/groupedbaryw.json

```diff
@@ -0,0 +1,15 @@
+[{"id":0,"type":132,"xval":"Aug 20","yval":74.928,"name":"Total Average","breakdown":"1634","breakdowntype":"YW","parentrefid":163,"parentreftype":135,"zval":1300},
```

Comment:
Actually, The string data set is for the customer journey, the date dataset I think is for reuse for some kind of mapping of a group of scores to dates. It'll be helpful probably to get the dates dataset working for because the chart we have already works with dates.